aspnetcore/mvc/controllers/routing.md

```diff
@@ -11,7 +11,7 @@ uid: mvc/controllers/routing
 By [Ryan Nowak](https://github.com/rynowak), [Kirk Larkin](https://twitter.com/serpent5), and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
 :::moniker range="< aspnetcore-6.0"
-[!INCLUDE [not-latest-version](~/includes/not-latest-version.md)]
+[!INCLUDE [not-latest-version](~/includes/not-latest-versionNM.md)]
 :::moniker-end
```

Comment:
@wadepickett @guardrex @tdykstra  The rare Not Moniker version for those few files that need it.